homeassistant/components/upnp/__init__.py

```diff
@@ -150,8 +157,24 @@ def device_discovered(info: Mapping[str, Any]) -> None:
         model=device.model_name,
```

Comment:
This device registry call, @StevenLooman what's the purpose of this?

Reply:
By the history it seems to be legacy code that can/should be removed, let's do that in a separate PR if that is the case :+1: